Frontend/android-native/fastlane/Fastfile

```
@@ -22,12 +22,31 @@ flavorsYP = ["yatriDriverProd"]
   lane :test do
     gradle(task: "test")
   end
 
+  desc "My awesome app"
+  lane :firebase_distribute do
+
+    gradle(
+      task: 'bundle' + 'nyUserProd' + 'Debug'
+    )
+    # build_android_app is a built-in fastlane action.
+    release = firebase_app_distribution(
```

RuboCop (1.73)

[warning] Useless assignment to variable - `release`.

(Lint/UselessAssignment)